pkg/shoes/shoes.go

```diff
@@ -88,10 +88,10 @@ func (c *GRPCClient) AddInstance(ctx context.Context, runnerName, setupScript st
 	}
 	resp, err := c.client.AddInstance(ctx, req)
 	if err != nil {
-		return "", "", "", fmt.Errorf("failed to AddInstance: %w", err)
+		return "", "", "", datastore.ResourceType(0), fmt.Errorf("failed to AddInstance: %w", err)
```

Comment:
Please use `datastore.ResourceTypeUnknown`

Reply:
Fixed in https://github.com/whywaita/myshoes/pull/169/commits/5a641811fe6c4bd8e4c9ac8b62dc3991bb279e2d